### DISCUSSION TITLE
How to provide the correct path for the FileOpener?

### DISCUSSION THREAD

**Discussion by kfvv** [capawesome-team/capacitor-plugins#55] [Q&A] [answered]
When providing a path to a local file, i get the following error message while opening the file:

`E/FileOpenerPlugin: Failed to find configured root that contains /img/msapplication-icon-144x144.png`

My code for the fileOpener is:

```
import { FileOpener } from '@capawesome-team/capacitor-file-opener'
await FileOpener.openFile({
    path: '/img/msapplication-icon-144x144.png',
    mimeType: 'image/png',
})
```

When I try to use a path to a file that is hosted on our server:

```
import { FileOpener } from '@capawesome-team/capacitor-file-opener'
    await FileOpener.openFile({
      path: 'https://www.mycompany.com/-/my/path/file.pdf'
      mimeType: 'application/pdf',
    })
```
I get the following error:

`E/FileOpenerPlugin: Failed to find configured root that contains /https://www.mycompany.com/-/my/path/file.pdf`

What am I doing wrong?

**Comment by robingenz** [comment, marked as answer]
You can only open local files, not files on a web server. You have to download them first (e.g. using [`downloadFile(...)​`](https://capacitorjs.com/docs/apis/filesystem#downloadfile)) and then open them.

**Edit**: 

> When providing a path to a local file, i get the following error message while opening the file:
> `E/FileOpenerPlugin: Failed to find configured root that contains /img/msapplication-icon-144x144.png`

Make sure to configure Android correctly, see [Android](https://capawesome.io/plugins/file-opener/#android).

**Reply by robingenz** [reply]
I updated my answer.
